integrationTests/testProcessorNode.go

```diff
@@ -146,6 +146,7 @@ type TestProcessorNode struct {
 	OwnAccount *TestWalletAccount
 	NodeKeys   *TestKeyPair
 
+	ExportFolder  string
```

Comment:
not required in integration tests.

Reply:
required for defer. in order to cleanup.